### ISSUE TITLE
Bro irritating broo

### ISSUE THREAD

**Issue by Hareshkumar-Y** [Hareshkumar-Y/Jai-balaji-traders#1]
I think that I would just forget about the parallax and create the product page alone...

Or else we can take some time for other features and just concentrate on the product page for now...

@Rithish1827

**Comment by Rithish1806** [comment]
Yeah bro sure ,we should just concentrate on the product page ,and complete
the work as soon as possible ,and later we can add some other features..

**Comment by Hareshkumar-Y** [comment]
Bro the product page is ready for the given products

@Rithish1827

**Comment by Hareshkumar-Y** [comment]
@Rithish1827 how is your work going??

**Comment by Hareshkumar-Y** [comment]
@Rithish1827 hi broo

**Comment by Rithish1806** [comment]
Solu bro

**Comment by Hareshkumar-Y** [comment]
Edhavadhu pannalaam brooo
